extension/controller/translation/translationWorker.js

```diff
@@ -302,7 +302,8 @@ class TranslationHelper {
         // instantiate the Translation Service
         constructTranslationService() {
             if (!this.translationService) {
-                let translationServiceConfig = { cacheSize: 10 };
+                // caching is disabled (see https://github.com/mozilla/firefox-translations/issues/288)
+                let translationServiceConfig = { cacheSize: 0 };
```

Comment:
I was wondering whether `0` would result in a fallback to a non-zero value, which is fortunately not the case:
https://github.com/mozilla/bergamot-translator/blob/5ae1b1ebb3fa9a3eabed8a64ca6798154bd486eb/src/translator/service.cpp#L34-L43

hence r+